<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QDialog" name="AuthRegForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>start - Car's-Specific</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::SystemLockScreen"/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_new_password">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>230</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pass_new">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/dom/image/1.png);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwordcheck">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mail">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 700 italic 11pt &quot;Segoe UI&quot;;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Reg">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>650</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_toReg">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>640</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Auth">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>610</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showPassword">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>pushButton_new_password</zorder>
  <zorder>pushButton_pass_new</zorder>
  <zorder>lineEdit_login</zorder>
  <zorder>lineEdit_password</zorder>
  <zorder>lineEdit_passwordcheck</zorder>
  <zorder>lineEdit_mail</zorder>
  <zorder>pushButton_Reg</zorder>
  <zorder>pushButton_toReg</zorder>
  <zorder>pushButton_Auth</zorder>
  <zorder>pushButton_showPassword</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
